lib/Components/Header/header.css

```diff
@@ -71,11 +71,11 @@
     .sm-nav-new{
         padding: 0;
         font-size: 1.3rem;
+        grid-template-columns: repeat(2,1fr);
+
```

Comment:
Unused space here - I believe that there is no need for a new line inside CSS declaration unless we are grouping certain related CSS properties. Example - 

This one is more readable -
.card {
  display: flex;
  flex-direction: column;
  justify-content: center;

  font-size: 2rem;
  font-weight: 600;
  font-family: 'serif'

  color: white;
  background-color: red
}

This is not readable - 
.card {
  justify-content: center;
  display: flex;
  flex-direction: column;
  font-size: 2rem;
  color: white;
  font-weight: 600;
  background-color: red;
  font-family: 'serif'
}